docs/modelserving/logger/logger.md

```diff
@@ -24,6 +24,9 @@ spec:
 kubectl create -f message-dumper.yaml
 ```
 
+!!! Note
+    You can create your own message-dumper service, in order to print during testing. Knative is not necessary for this feature.
+
```

Comment:
@johnzheng1975 I think it is better to just change to not use knative service as an example here

Reply:
I will add later.